packages/plugins/IconMenu/README.md

```diff
@@ -11,40 +11,70 @@ _Please note that the average user has never seen the client before and just wan
 
 ## Configuration
 
+### iconMenu
+
 > ⚠️ This plugin can't be configured with the `mapConfiguration`, but is configured during client construction. Currently, no way exists to add plugins to it after the build took place.
 
 For details on the `displayComponent` attribute, refer to the [Global Plugin Parameters](../../core/README.md#global-plugin-parameters) section of `@polar/core`.
 
-Usage during client build:
+| fieldName | type | description |
+| - | - | - |
+| initiallyOpen  | string?   | Id of the plugin which should be open on start; only applicable if the device doesn't have a small display  |
+| menus | Array | Defines which plugins should be rendered as part of the icon menu. |
+| layoutTag | enum NineLayoutTag | Defines where the icon menu is rendered. Only use `NineLayoutTag.TOP_RIGHT` here for the time being. |
+
+Use the configuration during client build.
+
+Example Configuration:
+```js
+{
+  initiallyOpen: 'layerChooser',
+  displayComponent: true,
+  menus: [
+    {
+      plugin: PolarPluginLayerChooser({}),
+      icon: 'fa-layer-group',
+      id: 'layerChooser',
+    },
+    {
+      plugin: PolarPluginDraw({}),
+      icon: 'fa-pencil',
+      id: 'draw',
+      hint: 'Draw or write something on the map'
+    },
+  ],
+  layoutTag: NineLayoutTag.TOP_RIGHT,
+}
+```
+
+### iconMenu.menus
+
+| fieldName | type | description |
+| - | - | - |
+| plugin | polar plugin | The plugin that should be part of the icon menu |
```

Comment:
@dopenguin Technically, the type is something like `(o: PluginOptions) => (v: Vue) => void`, but that's not exactly helpful, especially not for any kind of configuration automation/support tool that may or may not exist in the future. What, do you think, should we write here? I'd suggest `PolarPlugin` (to have it as we type our TS types) and call it a day.

Reply:
f536941285d77b6098b3d622547a8a24d77d1db0

I used your suggestion 'PolarPlugin'